controllers/ApiController.php

```diff
@@ -482,7 +482,7 @@ public function init($extra) {
 		
 		// Sorting by Item Type or Added By currently require writing to shard tables, so don't
 		// send those to the read replicas
-		if ($this->queryParams['sort'] == 'itemType' || $this->queryParams['sort'] == 'addedBy') {
+		if ($this->queryParams['sort'] == 'itemType' || $this->queryParams['sort'] == 'addedBy' || $this->queryParams['sort'] == 'editedBy') {
```

Comment:
We can switch this to `in_array($this->queryParams['sort'], ['itemType', 'addedBy', 'editedBy'])`